dist/pptxgen.es.js

```diff
@@ -942,6 +942,14 @@ function correctShadowOptions(ShadowProps) {
         // B: ROBUST: Cast any type of valid arg to int: '12', 12.3, etc. -> 12
         ShadowProps.opacity = Number(ShadowProps.opacity);
     }
+    // OPT: `color`
+    if (ShadowProps.color) {
+        // INCORRECT FORMAT
+        if (ShadowProps.color.startsWith('#')) {
+            console.warn('Warning: shadow.color should not include hash (#) character');

```

Comment:
This warning does not really tell me what I should use! Might be better to include an example of a hex color without the hash.